projects/jackson-databind/AdaLObjectReader3Fuzzer.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+max_len = 1500000
+len_control = 0
```

Comment:
@AdamKorcz @DavidKorczynski From now on can you put seed corpuses in other repos please? Putting them in here slows down cloning this repo and increases the space it takes forever.

Reply:
Sure! Have removed the corpus: https://github.com/google/oss-fuzz/pull/8521